litellm/proxy/_new_secret_config.yaml

```diff
@@ -1,5 +1,29 @@
+general_settings:
+  store_model_in_db: true
+  database_connection_pool_limit: 20
+  store_prompts_in_spend_logs: true
+  maximum_spend_logs_retention_period: "14d"
+
 model_list:
-  - model_name: openai-test
+  - model_name: fake-openai-endpoint
     litellm_params:
-      model: gpt-3.5-turbo
-      api_key: os.environ/OPENAI_API_KEY
\ No newline at end of file
+      model: openai/my-fake-model
+      api_key: my-fake-key
+      api_base: https://exampleopenaiendpoint-production.up.railway.app/
+  - model_name: bedrock-thinking-us.anthropic.claude-3-7-sonnet-20250219-v1:0
+    litellm_params:
+      model: bedrock/us.anthropic.claude-3-7-sonnet-20250219-v1:0
+      thinking: {"type": "enabled", "budget_tokens": 1024}
+      max_tokens: 1080
+      merge_reasoning_content_in_choices: true
+
+litellm_settings:
+  return_response_headers: true
+  store_audit_logs: true
+  callbacks: ["prometheus", "resend_email", "otel"]
+  cache: true
+  cache_params:
+    type: "redis"
+    host: "redis-18438.c277.us-east-1-3.ec2.redns.redis-cloud.com"
+    port: 18438
+    password: "hB44ThYlB4W4m7wpCUwrSzteHqvDKnDV"
```

Comment:
Friendly Advise:  You Redis server password is in the open !! Please change the password and put in the github secret.

Reply:
cc @ishaan-jaff